skuba-update/Makefile

```diff
@@ -1,17 +1,5 @@
 .PHONY: all
-all: install
-
-.PHONY: install
-install:
-	./setup.py install
```

Comment:
If developers like using `make` as the interface for installing why change that?

Reply:
because that's the wrong command?

suse-package/suse-changelog are not working, proving it's not used by devs, and has rotten (it's a duplicate from the time skuba and skuba-update were separate).

For install, I suppose ``pip install -e .`` should be used instead ?